Libraries/src/Amazon.Lambda.DynamoDBEvents/DynamoDBTimeWindowEvent.cs

```diff
@@ -0,0 +1,66 @@
+namespace Amazon.Lambda.DynamoDBEvents
+{
+    using System;
+    using System.Collections.Generic;
+
+#if !NETSTANDARD2_0
```

Comment:
Instead of using the `!` operator just say `NETCOREAPP_3_1` like every else in our libraries. Pretty soon we will need to do a sweep across all of our libraries to change all references of `NETCOREAPP_3_1` to `NETCOREAPP3_1_OR_GREATER` and it will be easier if we are consistent with are preprocessor usage. I don't want to do that work because I need to double check the .NET SDK version running in the build system to ensure it is new enough that it has the `GREATER` preprocessor symbols.

Reply:
I have changed it to `NETCOREAPP3_1_OR_GREATER` (else it was failing test case) since test case was having this conditional compilation check. Tested locally.